### PR TITLE
Prevent users from inviting existing members

### PR DESCRIPTION
**Pull request by kewitz**

### CODE REVIEW

**Review by Betree — APPROVED** [review]
Nice check

File: server/graphql/v2/mutation/MemberInvitationMutations.js [hunk 1]
```diff
@@ -58,6 +58,17 @@ const memberInvitationMutations = {
...
+        where: {
+          role: args.role,
+          CollectiveId: account.id,
+          MemberCollectiveId: memberAccount.id,
+        },
+      });
+      if (existingMembership) {
+        throw new BadRequest('Invited user is already a member of the account');
+      }
+
```
Comment (znarf) [inline server/graphql/v2/mutation/MemberInvitationMutations.js:71]: Could you move this in the `MemberInvitation.invite` method?

**Review by znarf — CHANGES_REQUESTED** [review]
Change requested.

**Comment by znarf** [comment]
What issue is this fixing?

File: server/graphql/v2/mutation/MemberInvitationMutations.js (hunk 1 above)
Comment (snitin315) [reply server/graphql/v2/mutation/MemberInvitationMutations.js:71]: I believe this check is already present https://github.com/opencollective/opencollective-api/blob/b9aeb1685c08290987a6a67e8e5fe7801d96e261/server/models/MemberInvitation.js#L159-L180

Comment (kewitz) [reply server/graphql/v2/mutation/MemberInvitationMutations.js:71]: Good catch.
There's no issue attached to this, I caught this while demoing last Friday and added to my to-do list.

**Comment by kewitz** [comment]
Sorry for wasting your time. I believe what happened is that we had multiple users with the same name on staging when I tested during demos.

**Pull request closed without merging**